pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/rbac/leader_election_role_binding.go

```diff
@@ -43,6 +44,13 @@ func (f *LeaderElectionRoleBinding) SetTemplateDefaults() error {
 const leaderElectionRoleBindingTemplate = `apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
+  labels:
+    app.kubernetes.io/name: rolebinding
+    app.kubernetes.io/instance: leader-election-rolebinding
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/part-of: {{ .ProjectName }}
+    app.kubernetes.io/managed-by: k8s.io
```

Comment:
```suggestion
    app.kubernetes.io/managed-by: kustomize
```

Reply:
resolved in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/de5ad3beb0988a49f533ee8c1a8c87fd82e7bf02